ush/config_defaults.sh

```diff
@@ -634,8 +634,10 @@ i_use_2mT4B=0
 #    Minimum number of ensemble members required a hybrid GSI analysis 
 #
 HYBENSMEM_NMIN=80
-ANAVINFO_FN="anavinfo.fv3lam_hrrr"
-ENKF_ANAVINFO_FN="anavinfo.enkf_fv3lam_hrrr"
+ANAVINFO_FN="anavinfo.rrfs"
+ANAVINFO_DBZ_FN="anavinfo.rrfs_dbz"
+ENKF_ANAVINFO_FN="anavinfo.enkf.rrfs"
+ENKF_ANAVINFO_DBZ_FN="anavinfo.enkf.rrfs_dbz"
```

Comment:
There are much more differences from anavinfo.enkf.rrfs_dbz vs anavinfo.rrfs_dbz than those from anavinfo.enkf.rrfs vs anavinfo.rrfs.  I am curious why it is so and why "t2m"/"q2m" are removed from the "met_guess" section?

Reply:
The anavinfo.rrfs_dbz is used for the GSI observer to process conventional and reflectivity observations at the same time.  The anavinfo.enkf.rrfs_dbz is used for the EnKF assimilation of reflectivity observations.  In the latter case, we don't need the t2m and q2m for observation operators, and therefore they aren't included in the met_guess section.